src/category_theory/category/Cat.lean

```diff
@@ -49,6 +50,13 @@ def objects : Cat.{v u} ⥤ Type u :=
 { obj := bundled.α,
   map := λ C D F, F.obj }
 
+/-- Any isomorphism in `Cat` induces an equivalence of the underlying categories. -/
+def equiv_of_iso {C D : Cat} (γ : C ≅ D) : C.α ≌ D.α :=
+{ functor := γ.hom,
+  inverse := γ.inv,
+  unit_iso := eq_to_iso $ eq.symm γ.hom_inv_id',
+  counit_iso := eq_to_iso γ.inv_hom_id' }
+
```

Comment:
There's a missing coercion from `Cat` to `Type` in this file.

Can I suggest replacing this whole namespace block with:
```
namespace Cat

instance : inhabited Cat := ⟨⟨Type u, category_theory.types⟩⟩

instance : has_coe_to_sort Cat :=
{ S := Type u,
  coe := bundled.α }

instance str (C : Cat.{v u}) : category.{v u} C := C.str

/-- Construct a bundled `Cat` from the underlying type and the typeclass. -/
def of (C : Type u) [category.{v} C] : Cat.{v u} := bundled.of C

/-- Category structure on `Cat` -/
instance category : large_category.{max v u} Cat.{v u} :=
{ hom := λ C D, C ⥤ D,
  id := λ C, 𝟭 C,
  comp := λ C D E F G, F ⋙ G,
  id_comp' := λ C D F, by cases F; refl,
  comp_id' := λ C D F, by cases F; refl,
  assoc' := by intros; refl }

/-- Functor that gets the set of objects of a category. It is not
called `forget`, because it is not a faithful functor. -/
def objects : Cat.{v u} ⥤ Type u :=
{ obj := λ C, C,
  map := λ C D F, F.obj }

/-- Any isomorphism in `Cat` induces an equivalence of the underlying categories. -/
def equiv_of_iso {C D : Cat} (γ : C ≅ D) : C ≌ D :=
{ functor := γ.hom,
  inverse := γ.inv,
  unit_iso := eq_to_iso $ eq.symm γ.hom_inv_id',
  counit_iso := eq_to_iso γ.inv_hom_id' }

end Cat
```

Reply:
Done!

Reply:
Ah, you didn't actually use my whole suggestion. I changed all the definitions to use the coercion rather than the `.α` projections. This isn't a big deal, but will potentially get in the way of syntactic matching further down the line. Better to write everything the same way. (We should possibly even add a simp lemma rewriting C.α as the coercion.)

Reply:
Oops! I'll change this later today when I get a chance.

Reply:
Okay, I think I got them all.